src/components/ScaleLattice.vue

```diff
@@ -0,0 +1,340 @@
+otonalFundamental
+<script setup lang="ts">
+import type { Interval, Scale } from "scale-workshop-core";
+import {
+  computed,
+  onMounted,
+  type ComputedRef,
+  ref,
+  onBeforeUnmount,
+} from "vue";
+import { PRIMES } from "xen-dev-utils";
+import { monzoEuclideanDistance, getPairCombinations } from "@/utils";
+const props = defineProps<{
+  scale: Scale;
+  width: number;
+  height: number;
+  heldScaleDegrees: Set<number>;
+}>();
+
+type Coord = { x: number; y: number };
+type Node = {
+  x: number;
+  y: number;
+  label: string;
+  index: number;
+  fill: string;
+};
+
+type Coords = {
+  [key: string]: Coord;
+};
+
+const coords: Coords = {
+  "1": {
+    x: 0,
+    y: 0,
+  },
+  "2": {
+    x: 0,
+    y: 0,
+  },
+  "3": {
+    x: 40,
+    y: 0,
+  },
+  "5": {
+    x: 0,
+    y: -40,
+  },
+  "7": {
+    x: 13,
+    y: -11,
+  },
+  "11": {
+    x: -14,
+    y: -18,
+  },
+  "13": {
+    x: -8,
+    y: -4,
+  },
+  "17": {
+    x: -5,
+    y: -32,
+  },
+  "19": {
+    x: 7,
+    y: -25,
+  },
+  "23": {
```

Comment:
Scale Workshop goes up to 97. Please make an issue about supporting the full range if you don't want to do all primes in this PR.

Reply:
I don't think it makes sense to predefine all coordinates. In my opinion, a UI so that a user can add, remove and change coords for the primes they need would be more useful.

Reply:
Sure, we can do tiered issues. One for completing the predefined coords and another for adding user-configurability. Just let me know if you're strongly against predefinition beyond the 23-limit. To me, going up to 97-limit is an easy win and better than nothing.

Reply:
Hmm I really don't have the time to figure out a good set of coordinates, I believe it requires much experimentation and a good sense of which sort of scales are actually going to be made, otherwise it's just randomness. 

Also as you can see here, the coordinate system is already quite cluttered and we only have 9 primes, adding 15 more seems like it'd be pretty hard to understand and distinguish between one or the other.
![image](https://github.com/xenharmonic-devs/scale-workshop/assets/14334071/b663fea8-f700-4467-b7db-6a91ff04499c)


But if we wanted to make a default set, I believe that a community effort would be a better fit, if possible.  Perhaps opening a call for suggestions and then a round of votes by members of the community would  get us somewhere.

In that case adding user-configurability first would allow them to easily explore and propose something.